AlphaWallet/Market/Coordinators/UniversalLinkInPasteboardCoordinator.swift

```diff
@@ -12,9 +12,9 @@ class UniversalLinkInPasteboardCoordinator: Coordinator {
     
     func start() {
         guard let contents = UIPasteboard.general.string?.trimmed else { return }
-        guard contents.hasPrefix(UniversalLinkHandler().urlPrefix) else { return }
-        guard contents.count > UniversalLinkHandler().urlPrefix.count else { return }
         guard let url = URL(string: contents) else { return }
+        guard contents.hasPrefix(UniversalLinkHandler().urlPrefix)
+                      || contents.hasPrefix(Constants.legacyMagicLinkPrefix) else { return }
```

Comment:
This means it will only auto-detect MagicLinks for the existing network. Which is fine, if that's the intention.

Reply:
Hmm, might be best to port the logic which checks displays an error if a correct link on the wrong network

Reply:
@James-Sangalli yeah, for completeness.